embedchain/embedchain.py

```diff
@@ -541,17 +541,11 @@ def set_collection_name(self, name: str):
         # TODO: Check whether it is necessary to assign to the `self.collection` attribute,
         # since the main purpose is the creation.
 
-    def count(self) -> int:
-        """
-        Count the number of embeddings.
-
-        DEPRECATED IN FAVOR OF `db.count()`
-
-        :return: The number of embeddings.
-        :rtype: int
-        """
-        logging.warning("DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.")
+    def db.count(self) -> int:
```

Comment:
What is the reason for doing this? I am not sure if this is needed. You may just want to remove this function altogether I think.